fix(wikiwand): syntax highlighting

somerand0mcat opened the pull request:
## 🔧 What does this fix? 🔧
fixes syntax highlighting aswell as some borders too

<details>
<summary>before/after</summary>
<img src="https://github.com/user-attachments/assets/84345f37-6988-4110-9ce7-7fea3d939919">
<hr>
<img src="https://github.com/user-attachments/assets/a9bde0a1-e445-4fbf-be94-50a1ba09c5a7">
</details>

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

uncenter commented:
This doesn't match our syntax highlighting guidelines, looks like you just made their colors Catppuccinified. Try referencing our other editor highlighting ports.

somerand0mcat commented:
> This doesn't match our syntax highlighting guidelines, looks like you just made their colors Catppuccinified. Try referencing our other editor highlighting ports.

can i get the link to these syntax highlighting guidelines?

somerand0mcat commented:
> This doesn't match our syntax highlighting guidelines, looks like you just made their colors Catppuccinified. Try referencing our other editor highlighting ports.

@uncenter i've changed the syntax highlighting based off some userstyles, i've also updated the screenshot

uncenter commented:
Still doesn't look right, this is what I'm getting with Shiki/VS Code:
![CleanShot 2024-08-14 at 16 03 11@2x](https://github.com/user-attachments/assets/94a4ec8c-64d1-480e-90fe-ae572ee2886a)

somerand0mcat commented:
does this work?
![image](https://github.com/user-attachments/assets/c707fefa-1a40-476e-8b7a-f9f562c816ab)

somerand0mcat commented:
> Looks better but let's just use https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#pygments lol.

i just put in the code for the pygments and the syntax highlighting instantly gets themed lol

somerand0mcat commented:
done

uncenter commented on styles/wikiwand/catppuccin.user.css:
```diff
@@ -368,6 +398,9 @@
     a.wl {
       color: @accent-color !important;
     }
+    .wm_content__0bruk .mw-highlight, pre {
+      border-color: @surface2 !important;
+    }
```
```suggestion
    pre, code {
      border-color: @surface2 !important;
    }
```

uncenter commented on styles/wikiwand/catppuccin.user.css:
```diff
@@ -72,6 +101,7 @@
     --color-bg: @base;
     --color-text: @text;
     --color-grey: @subtext0;
+    --color-subtle: @subtext0;
```
What does this theme?

somerand0mcat replied:
themes this
![image](https://github.com/user-attachments/assets/8fe4f8cb-447a-4abd-9ee1-47a61c5873f7)

uncenter replied:
Mind sending me the link you found it on? Just want to see how it looks in context!

somerand0mcat replied:
sure, https://www.wikiwand.com/en/%22Hello,_World!%22_program

uncenter reviewed (commented):
Good job on adding the Pygments syntax highlighting, just these two comments :)

uncenter commented on styles/wikiwand/catppuccin.user.css:
```diff
@@ -368,6 +398,9 @@
     a.wl {
       color: @accent-color !important;
     }
+    pre, code {
```
```suggestion
    pre, code, samp, kbd {
```
From your example above actually.

uncenter commented on styles/wikiwand/catppuccin.user.css:
Why was this removed?

somerand0mcat replied:
i thought we didn't need the variable (i was really tired)

uncenter reviewed (approved):
Great work! Looks wayy better now. And inline code blocks are still unthemed, but that should be fixed for us by https://github.com/catppuccin/python/pull/69.
